stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -187,16 +195,11 @@ internal class ChatDomainImpl internal constructor(
         _online.value = false
         _totalUnreadCount.value = 0
         _channelUnreadCount.value = 0
-        _errorEvent.value = null
```

Comment:
Lint (as in Android Lint, not Ktlint) was failing on this line, we're just not running it very often. Can't set a `null` value in a `LiveData` that has a non-nullable type parameter. We don't really need to reset it though, because we're using it for events.